### PR TITLE
KE-1054 implement tag list contains filter

### PR DESCRIPTION
**Pull request by KaczuH**
Fixes #<issue>

## Checklist:
- [x] My code follows the pykechain style
    - [x] I added type hinting to all functions
    - [x] I added documentation for all public functions
    - [x] I locally used `tox` to check for dists and docs errors
    - [x] My code passes the `pep8` and `flake8` linting checks and no warnings
    - [x] I removed unused imports, using `Code > Optimize Imports` in PyCharm
    - [x] I used [`black`](https://github.com/psf/black) to format the new code
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] I committed fresh test cassettes
    - [x] I have proper test coverage (don't decline the coverage of the test)
- [x] I asked another teammate to review the code
- [x] I update the Changelog.md with the appropriate changes.

### CODE REVIEW

File: tests/test_contexts.py [hunk 1]
```diff
@@ -54,6 +54,20 @@ def test_retrieve_contexts_via_client_using_scope_filter(self):
             contexts = self.client.contexts(scope=str(self.project.id))
             self.assertIsInstance(contexts[0], Context)
 
+    def test_retrieve_contexts_via_client_using_tags_list__contains_filter(self):
```
Comment (raduiordache) [inline tests/test_contexts.py:57]: For me this test fails when I run it in Pycharm. It returns a `ConnectionError`.

**Review by raduiordache — CHANGES_REQUESTED** [review]
There are a few tests failing. Can you make sure all tests are passing first, please?

**Comment by jberends** [comment]
what is the sense of this PR? I only see an additional test in the changes. What was the original question to solve?

**Review by raduiordache — COMMENTED** [review]
I see only one test added. Does it mean that the filtering worked by default and needed no extra implementation?

**Review by raduiordache — CHANGES_REQUESTED** [review]
Don't forget to follow the checklist and add a Changelog message.

**Comment by KaczuH** [comment]
@jberends  @raduiordache The filtering worked out of the box so i just added a test for it. Did i miss something?

**Comment by jberends** [comment]
> @jberends @raduiordache The filtering worked out of the box so i just added a test for it. Did i miss something?

no.

please add changelog

**Review by jberends — APPROVED** [review]
- [ ] add changelog

**Comment by jberends** [comment]
Woop woop!! First merge!!

**Pull request merged**